api/api.go

```diff
@@ -142,6 +142,7 @@ func (a *ApplicationHandler) BuildRoutes() *chi.Mux {
 						// TODO(all): should the InstrumentPath change?
 						eventRouter.With(middleware.InstrumentPath("/events")).Post("/", handler.CreateEndpointEvent)
 						eventRouter.Post("/fanout", handler.CreateEndpointFanoutEvent)
+						eventRouter.Post("/broadcast", handler.CreateBraodcastEvent)
```

Comment:
@danvixent, I think there's a typo here. The handler name should be `CreateBroadcastEvent`.